XboxKeyboardMouse/XboxController/Emulation/XboxStream.cs

```diff
@@ -43,7 +90,7 @@ public struct RECT {
                     IntPtr handle = GetForegroundWindow();
                     if (GetWindowText(handle, text, count) > 0)
                     {
-                        if (!text.ToString().Equals(XBOXAPP))
+                        if (!defaultAppNames.Contains(text.ToString()))
```

Comment:
I'm okay with the current solution, however, If every app has Xbox in the name - we could just match for the string "Xbox" rather than maintaining a list of window titles.

Reply:
Or a simple regex to match `Playstation|Xbox` would work.

Reply:
I wanted to avoid input locking say browser or XKM itself whose titles may have one of those keywords in it.